pom.xml

```diff
@@ -1,370 +1,280 @@
-            <build>
-                <plugins>
-                    <plugin>
-                        <artifactId>maven-surefire-plugin</artifactId>
-                        <configuration>
-                            <excludes>
-                                <exclude>%regex[.*container.*]</exclude>
-                            </excludes>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-release-plugin</artifactId>
+				<configuration>
+					<autoVersionSubmodules>true</autoVersionSubmodules>
+					<pushChanges>false</pushChanges>
+					<localCheckout>true</localCheckout>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<systemProperties>
+						<!-- <jboss.version>${version.wildfly}</jboss.version> -->
+						<path.tools_jar>${path.tools_jar}</path.tools_jar>
+					</systemProperties>
+					<runOrder>alphabetical</runOrder>
+					<excludes>
+						<exclude>%regex[.*local.*]</exclude>
+					</excludes>
+				</configuration>
+			</plugin>
+			<!-- We need to jarjar this, since WildFly grabs all org.jboss.byteman 
+				as system packages -->
+			<plugin>
+				<groupId>org.sonatype.plugins</groupId>
+				<artifactId>jarjar-maven-plugin</artifactId>
+				<version>${version.jarjar}</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>jarjar</goal>
+						</goals>
+						<configuration>
+							<includes>
+								<include>org.jboss.byteman:byteman-submit</include>
+							</includes>
+							<rules>
+								<rule>
+									<pattern>org.jboss.byteman.**</pattern>
+									<result>org.jboss.arquillian.extension.byteman.@1</result>
+								</rule>
+							</rules>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
+
+	<profiles>
+		<profile>
+			<id>arq-wildfly-managed</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<arq.container.chameleon.configuration.chameleonTarget>wildfly:10.1.0.Final:managed
+								</arq.container.chameleon.configuration.chameleonTarget>
+							</systemPropertyVariables>
+							<runOrder>alphabetical</runOrder>
+							<excludes>
+								<exclude>%regex[.*local.*]</exclude>
+							</excludes>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+
+
+		<profile>
+			<id>jboss as</id>	
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<arq.container.chameleon.configuration.chameleonTarget>jboss
+									as:7.1.1.Final:managed
```

Comment:
This is because you run it against JDK 8. I think we can drop this profile anyway.